source/common/router/config_impl.h

```diff
@@ -358,8 +358,8 @@ class RouteEntryImplBase : public RouteEntry,
 
   void finalizePathHeader(Http::HeaderMap& headers, const std::string& matched_path,
                           bool insert_envoy_original_path) const;
-  const HeaderParser& requestHeaderParser() const { return *request_headers_parser_; };
-  const HeaderParser& responseHeaderParser() const { return *response_headers_parser_; };
+  const HeaderParser& requestHeaderParser() const { return *route_action_request_headers_parser_; };
+  const HeaderParser& responseHeaderParser() const { return *route_action_response_headers_parser_; };
```

Comment:
This change for these two getters is reasonable since we used to work on `RouteAction` level. Do you think we need to have getters for the `Route` level too?